WordPress/WordPressTest/ReaderSiteServiceTests.swift

```diff
@@ -1,20 +1,14 @@
-import XCTest
+import Nimble
 import OHHTTPStubs
+@testable import WordPress
+import WordPressKit
+import XCTest
 
 class ReaderSiteServiceTests: CoreDataTestCase {
 
-    private var service: ReaderSiteService!
-
-    override func setUp() {
-        let accountService = AccountService(coreDataStack: contextManager)
-        accountService.createOrUpdateAccount(withUsername: "testuser", authToken: "authtoken")
-        self.service = ReaderSiteService(coreDataStack: contextManager)
-
-        stub(condition: isHost("public-api.wordpress.com")) { request in
-            NSLog("[Warning] Received an unexpected request sent to \(String(describing: request.url))")
-            return HTTPStubsResponse(error: URLError(.notConnectedToInternet))
-        }
+    override class func tearDown() {
         HTTPStubs.removeAllStubs()
+        super.tearDown()
     }
```

Comment:
Admittedly, these changes weren't necessary. But it's my firm belief that tests are better off without local state (instance properties) both to help local reasoning and because of XCTest odd memory management where all test cases remain alive in memory until the whole suite has finished.

Also notice that the call to `stub` was a no-op because it was immediately followed by `removeAllStubs()`